src/graphql/typeDefs.graphql

```diff
@@ -654,10 +654,19 @@ type MonsterAction {
   usage: Usage
 }
 
+type MonsterArmorClass {
+  type: String!
+  desc: String
+  value: Int!
+  armor: [IEquipment]
```

Comment:
There's an `Armor` GraphQL type. I think it would make more sense to use it here than `IEquipment` because it's always going to be armor.